client/src/main/java/io/pravega/client/stream/impl/StreamCut.java

```diff
@@ -29,8 +29,10 @@
 @Data
 public class StreamCut implements Serializable {
 
+    @NonNull
     private final Stream stream;
-    @Getter(value = AccessLevel.PACKAGE)
+    @Getter
```

Comment:
This should not be public. If this is needed to access cross package, then you may want to model it on EventPointer which works around this with a double interface.

Reply:
Will fix this as part of #2006